src/Tribe/Views/V2/Widgets/Service_Provider.php

```diff
@@ -158,6 +167,97 @@ public function enable_widget_copy_paste( $result, $server, $request ) {
 		return $result;
 	}
 
+	/**
+	 * Enable widget copy paste for the Legacy Widgets that we are registering.
+	 * 
+	 * @since 6.11.2
+	 * 
+	 * @param mixed           $result  The result of the rest request.
+	 * @param WP_REST_Request $request The REST request.
+	 * @param string          $route   The route of the request.
+	 * @param array           $handler The handler of the request.
+	 * 
+	 * @return mixed The result of the rest request.
+	 */
+	public function enable_saving_widget_copied( $result, $request, $route, $handler ) {
```

Comment:
Is `$handler` used in this method? I don't see it used, so I think we can drop it from the param list.